libcob/common.h

```diff
@@ -1369,7 +1369,9 @@ typedef struct __cob_module {
 #define COB_DUMP_LS		0x20  		/* LINKAGE SECTION */
 #define COB_DUMP_LO		0x40  		/* LOCAL-STORAGE SECTION */
 #define COB_DUMP_ALL	(COB_DUMP_FD|COB_DUMP_WS|COB_DUMP_RD|COB_DUMP_SD|COB_DUMP_SC|COB_DUMP_LS|COB_DUMP_LO)
-
+#if 0
+	unsigned char		unused[1];		/* Use these flags up later, added for alignment */
+#endif
```

Comment:
please check if this is the correct padding in 4.x and either enable that (possibly adjusting the padding size) or, if that's already well padded, drop the code

Reply:
Looks aligned at this point. Though there are other fields that act as flags at the end of the struct that could be moved upwards. Maybe we should plan a cleanup/reordering of these structs once all patches have been merged from GC3 to GC4 ?

Reply:
Feel free to reorder that in 4-x at any time (maybe directly when doing the merge); it is likely useful to keep some "reserve" for additional flags as well.
In any case I'd suggest to move the code above to the end of the flags and directly have some reserved space instead of the `#if 0` deactivation.